### DISCUSSION TITLE
Add a Pre-Bury Function to Sopport AnkiDroid Users

### DISCUSSION THREAD

**Discussion by danny-jordan** [mortii/anki-morphs#311] [Ideas]
First of all, thanks for your amazing work on making this add on available again. I really love it!

Currently, the AnkiMorph add-on allows filtering sentence banks by the number of unknown morphs, suspending cards with zero unknown morphs, and ranking morphs by frequency lists. All these adjustments apply immediately after pressing the "Recalc" button in Anki Desktop. However, the key runtime feature—the "Bury" function that skips duplicate morph cards containing the same unknown morph already seen today—is only active on Anki Desktop. Since AnkiDroid does not support plugins, this runtime burying does not occur on mobile, forcing users to manually skip duplicates. This is near impossible to deal with for large sentence decks.

To address this, I propose adding a "Pre-Bury" function in the setting menu. When the user hits "Recalc," this feature would automatically identify duplicate cards sharing the same unknown morph and bury all but the first one. By pre-burying duplicates, the filtered deck would remain optimized for review on AnkiDroid, effectively bringing the runtime behavior of the desktop environment to mobile sessions, and removing the need for manual skipping.
In case the pre selected first card is shit, we can suspend it manually. The next day we would see the second card since the morph was not learnt, and 2nd card is now first.

Let me know what you think about it.

**Comment by mortii** [comment]
This basically already exists, it's the ['shift new cards that are not the first to have the unknown morph [...]'](https://mortii.github.io/anki-morphs/user_guide/setup/settings/card_handling.html) option. It doesn't bury the cards, but it functionally achieves the same thing.

**Comment by danny-jordan** [comment]
Sorry for the inconvenience. I checked the documentation but didn't recognized this function does basically the same. Thats amazing! Thanks alot for your quick reply
